docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -1422,9 +1454,60 @@ Wave III
   * - Model
     - Outputs
     - Overall strata
-  * - 14
-    - 
-    - 
+  * - 14.0
+    - 1. Deaths and YLLs (cause-specific)
+      2. YLDs (cause-specific)
+      3. Cause state person time
+      4. Cause state transition counts
+      5. Stunting state person time, stratified by antenatal intervention coverage
+      6. Wasting state person time, stratified by antenatal intervention coverage
+    - * Age group
+      * Sex
+  * - 15.0
+    - 1. Deaths and YLLs (cause-specific)
+      2. YLDs (cause-specific)
+      3. Cause state person time
+      4. Cause state transition counts
+      5. Stunting state person time, stratified by antenatal intervention coverage
+      6. Wasting state person time, stratified by antenatal intervention coverage
```

Comment:
Same thing -- let's add underweight and wasting transition observers and I think we can drop antenatal coverage stratification